Makefile

```diff
@@ -39,6 +39,7 @@ production: production-cluster
 
 ci:
 	$(eval export AUTO_APPROVE=-auto-approve)
+	$(eval include global_config/dns-domain.sh)
```

Comment:
this shouldn't be added to ci. This section is just for flags to enable cicd workflows